Add umbraco commerce product feed package (#6007)
umbraco/UmbracoDocs: commerce-add-ons/packages/product-feeds/README.md

Reviewer: I would suggest:

"Need to generate Umbraco Commerce product feed? There's a plugin for that."

"After building up your commerce website, you can expand your product reach and show your products to a larger audience with product feeds. Umbraco Commerce Product Feed is a free and open-sourced add-on for Umbraco Commerce that helps you generate the XML files that are used in services like Google Merchant Center."

Reviewer 2: I agree that "After building up your commerce website, you can expand your product reach and show your products to a larger audience with product feeds. Umbraco Commerce Product Feed is a free and open-sourced add-on for Umbraco Commerce that helps you generate the XML files that are used in services like Google Merchant Center." can be added after the description 👍

---
description: >-
  This is the documentation for the Product Feed package for Umbraco Commerce.
---

## Introduction

> After building an e-commerce website, you might want to share your products with an external service like Google Merchant Center to reach more viewers.

Reviewer 2: I like the suggestion made by Bolette as an intro:
```suggestion
After building up your commerce website, you can expand your product reach and show your products to a larger audience with product feeds. Umbraco Commerce Product Feed is a free and open-sourced add-on for Umbraco Commerce that helps you generate the XML files that are used in services like Google Merchant Center.
```

> ## Introduction
> After building an e-commerce website, you might want to share your products with an external service like Google Merchant Center to reach more viewers.

Reviewer 2: I like the suggestion made by Bolette as an intro, also because it is written that is open-source which the 2 other packages mention as well:
```suggestion
After building up your commerce website, you can expand your product reach and show your products to a larger audience with product feeds. Umbraco Commerce Product Feed is a free and open-sourced add-on for Umbraco Commerce that helps you generate the XML files that are used in services like Google Merchant Center.
```

`Umbraco Commerce Product Feeds` package is an XML feed generator plugin that simplifies the creation of structured XML files. It automatically extracts product data from Umbraco nodes, organizes it according to standards and also allows you to add more information to the feed.

Current supported feed template:
- `Google Merchant Center` feed template.

> 

Reviewer 2: missing content that the other 2 packages have as well. 
```suggestion
In this section, we will guide you through the key steps necessary to get you started with the Product Feeds package for Umbraco Commerce.

It is assumed that before we begin you already have an Umbraco website configured and Umbraco Commerce installed and a store set up. If you are not at this stage yet, please read the [core Umbraco Commerce documentation](https://docs.umbraco.com/umbraco-commerce/) to learn how to get started.
```

> ## Useful links

Reviewer 2: ```suggestion
# Useful links
```

---
description: >-
  This is the documentation for the Product Feed package for Umbraco Commerce.
---

## Introduction
After building an e-commerce website, you might want to share your products with an external service like Google Merchant Center to reach more viewers.

`Umbraco Commerce Product Feeds` package is an XML feed generator plugin that simplifies the creation of structured XML files. It automatically extracts product data from Umbraco nodes, organizes it according to standards and also allows you to add more information to the feed.

> Current supported feed template:
> - `Google Merchant Center` feed template.
> In this section, we will guide you through the key steps necessary to get you started with the Product Feeds package for Umbraco Commerce.

Reviewer 2: ```suggestion
Current supported feed template:
- `Google Merchant Center` feed template.

In this section, we will guide you through the key steps necessary to get you started with the Product Feeds package for Umbraco Commerce.
```

It is assumed that before we begin you already have an Umbraco website configured and Umbraco Commerce installed and a store set up. If you are not at this stage yet, please read the [core Umbraco Commerce documentation](https://docs.umbraco.com/umbraco-commerce/) to learn how to get started.
# Useful links

> - Package source code: https://github.com/umbraco/Umbraco.Commerce.ProductFeeds
> - Package issue tracker: https://github.com/umbraco/Umbraco.Commerce.ProductFeeds/issues

Reviewer 2: ```suggestion
- [Umbraco Commerce Product Feed source code](https://github.com/umbraco/Umbraco.Commerce.ProductFeeds)
- [Umbraco Commerce Product Feed issue tracker](https://github.com/umbraco/Umbraco.Commerce.ProductFeeds/issues)
```

---
description: >-
  This is the documentation for the Product Feed package for Umbraco Commerce.
---

## Introduction
After building an e-commerce website, you might want to share your products with an external service like Google Merchant Center to reach more viewers.

`Umbraco Commerce Product Feeds` package is an XML feed generator plugin that simplifies the creation of structured XML files. It automatically extracts product data from Umbraco node, organizes it according to standards and allows you to add more information to the feed.

The `Umbraco Commerce Product Feeds` package supports the Google Merchant Center feed template.

## Quick start
- Install the package from nuget: [![NuGet](https://img.shields.io/nuget/v/Umbraco.Commerce.ProductFeeds.svg?style=modern&label=nuget)](https://www.nuget.org/packages/Umbraco.Commerce.ProductFeeds/) 

- To add this plugin to your website, locate where you call `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()`.

```csharp
        umbracoBuilder.AddUmbracoCommerce(ucBuilder => {
            ucBuilder.AddCommerceProductFeeds(); // add this line
        }
```

> - After that, go to your backoffice, open your store's setting page then click on `Product Feed` section.
> ![product feed list page](./media/product-feed-list-page.png).
> 
> - Inside `Product Feeds` page, click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.
> 
> ![feed setting page](./media/feed-setting-page.png).
> 
> - After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.
> ![open feed link](./media/open-feed-link.png)

Reviewer 2: so the images are in new line from the content
```suggestion
- After that, go to your backoffice, open your store's setting page then click on `Product Feed` section.

![product feed list page](./media/product-feed-list-page.png).

- Inside `Product Feeds` page, click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.

![feed setting page](./media/feed-setting-page.png).

- After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.

![open feed link](./media/open-feed-link.png)
```

---
description: >-
  This is the documentation for the Product Feed package for Umbraco Commerce.
---

## Introduction

> After building an e-commerce website, you might want to share your products with an external service like Google Merchant Center to reach more viewers.

Reviewer 2: ```suggestion
After building up your commerce website, you can expand your product reach and show your products to a larger audience with product feeds. Umbraco Commerce Product Feed is a free and open-sourced add-on for Umbraco Commerce that helps you generate the XML files that are used in services like Google Merchant Center.
```

> `Umbraco Commerce Product Feeds` package is an XML feed generator plugin that simplifies the creation of structured XML files. It automatically extracts product data from Umbraco node, organizes it according to standards and allows you to add more information to the feed.
> 
> The `Umbraco Commerce Product Feeds` package supports the Google Merchant Center feed template.
> 

Reviewer 2: ```suggestion
adfdasfdsafasdafsfas

```

## Quick start
- Install the package from nuget: [![NuGet](https://img.shields.io/nuget/v/Umbraco.Commerce.ProductFeeds.svg?style=modern&label=nuget)](https://www.nuget.org/packages/Umbraco.Commerce.ProductFeeds/) 

- To add this plugin to your website, locate where you call `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()`.

```csharp
        umbracoBuilder.AddUmbracoCommerce(ucBuilder => {
            ucBuilder.AddCommerceProductFeeds(); // add this line
        }
```

> - After that, go to your backoffice, open your store's setting page then click on `Product Feed` section.
> 
> ![product feed list page](./media/product-feed-list-page.png).
> 
> - Inside `Product Feeds` page, click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.
> 
> ![feed setting page](./media/feed-setting-page.png).
> 
> - After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.
> 
> ![open feed link](./media/open-feed-link.png)
> 
> - `Google Merchant Center Feed` sample:
> ![google merchant center feed](./media/google-merchant-center-feed.png)

Reviewer 2: ```suggestion
3. Open your store's settings page in the backoffice.

4. Click on `Product Feed` section.

![product feed list page](./media/product-feed-list-page.png).

5. Click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.

![feed setting page](./media/feed-setting-page.png).

6. After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.

![open feed link](./media/open-feed-link.png)

`Google Merchant Center Feed` sample:

![google merchant center feed](./media/google-merchant-center-feed.png)
```

Author: suggestion was committed.

---
description: >-
  Need to generate umbraco commerce product feed? There's a plugin for that.
---

## Introduction
After building up your e-commerce website, it's quite common that you want to share your fancy products to a external service like Google Merchant Center to reach out to more viewers.

We have a XML feed generator plugin which simplifies the creation of structured XML files like that. It automatically extracts product data from Umbraco node, organizes it according to standards and allows you to add more information to the feed.

Checkout `Umbraco Commerce Product Feeds plugin`.

Supported feed templates
- Google Merchant Center.

> ## Quick start

Reviewer 2: 1. Please move this content into a new article called "Installation" to match with the other 2 plugins: https://docs.umbraco.com/umbraco-commerce-packages/checkout/installation 
2. Then you can add some content from the overview page of the other packages: https://docs.umbraco.com/umbraco-commerce-packages/deploy/deploy

Author: Suggestions were applied.

> - Install the package from nuget: [![NuGet](https://img.shields.io/nuget/v/Umbraco.Commerce.ProductFeeds.svg?style=modern&label=nuget)](https://www.nuget.org/packages/Umbraco.Commerce.ProductFeeds/) 

Author: I change the text as you suggested, but I prefer to keep the image to nuget because it shows the latest version and the status of the package.

> - To add this plugin to your website, locate where you call `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()`.

Reviewer 2: just to have more info on where this might be, added a link to https://docs.umbraco.com/umbraco-commerce/key-concepts/umbraco-commerce-builder#registering-dependencies 

Also, we start with actions first when giving instructions
```suggestion
2. Locate where you [register the dependencies](https://docs.umbraco.com/umbraco-commerce/key-concepts/umbraco-commerce-builder#registering-dependencies) `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()` to add this plugin to your website. 
```

```csharp
        umbracoBuilder.AddUmbracoCommerce(ucBuilder => {
            ucBuilder.AddCommerceProductFeeds(); // add this line
        }
```

> - After that, go to your backoffice, open your store's setting page then click on `Product Feed` section.
> ![product feed list page](./media/product-feed-list-page.png).
> 
> - Inside `Product Feeds` page, click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.
> 
> ![feed setting page](./media/feed-setting-page.png).
> 
> - After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.
> ![open feed link](./media/open-feed-link.png)
> 
> - `Google Merchant Center Feed` sample:

Reviewer 2: ```suggestion
3. Open your store's setting page in the backoffice 

4. Click on `Product Feed` section.

![product feed list page](./media/product-feed-list-page.png).

5. Click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.

![feed setting page](./media/feed-setting-page.png).

6. After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.

![open feed link](./media/open-feed-link.png)

 `Google Merchant Center Feed` sample:
```
